packages/cli/src/errors/shared-workflow-not-found.error.ts

```diff
@@ -1,3 +1,3 @@
-import { ApplicationError } from 'n8n-workflow';
+import { UnexpectedError } from 'n8n-workflow';
 
-export class SharedWorkflowNotFoundError extends ApplicationError {}
+export class SharedWorkflowNotFoundError extends UnexpectedError {}
```

Comment:
I believe this is a `UserError` since it fails if the input from user (workflow id) is not found